components/gantt/gantt-tree/templates/column-chooser.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Column Chooser
+page_title: Gantt - Column Chooser Template
+description: Customize the content of the Column Chooser.
+slug: gantt-templates-column-chooser
+tags: telerik,blazor,gantt,templates,column,menu
+published: True
+position: 40
+---
+
+# Column Chooser Template
+
+The `Column Chooser Template` lets you control the rendering of the [ColumnChooser]({%slug gantt-column-menu%}#column-chooser). It exposes a `context` object that contains a List of all columns in the Gantt. 
```

Comment:
"Column Chooser Template" doesn't appear to be a piece of code so use the default formatting instead of `inline code`.
```suggestion
The Column Chooser Template lets you control the rendering of the [ColumnChooser]({%slug gantt-column-menu%}#column-chooser). It exposes a `context` object that contains a list of all columns in the Gantt. 
```